product_supplierinfo_qty_multiplier/README.rst

```diff
@@ -0,0 +1,97 @@
+===================================
+Product supplierinfo qty multiplier
+===================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpurchase--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/purchase-workflow/tree/13.0/product_supplierinfo_qty_multiplier
+    :alt: OCA/purchase-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-product_supplierinfo_qty_multiplier
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/142/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to define multiplier quantities at the supplier level in the
+products to apply it in the purchase order lines.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Purchase > Products > Products.
+#. Go to Acoustic Bloc Screens product.
+#. Go to 'Purchase' tab and edit 'Wood Corner' line to set 'Multiplier Qty' to 2.0.
```

Comment:
Again, don't restrict it to one product, and warn the user that the column might be hidden.
```suggestion
#. Go to the 'Purchase' tab and edit a vendor pricelist line to set 'Multiplier Qty' to 2.0 (If you can't find that column, you can activate it by clicking the tree dots at the corner of the table and enabling it).
```